_data/trans/vi.json

```diff
@@ -25,9 +25,9 @@
     "mikerogers": "Mike Rogers",
     "name": "Tiếng Việt",
     "noinfo": "Không có thông tin",
-    "noresults": "Can't find what you're looking for?",
-    "noresultshelp": "Help make JustDeleteMe better",
-    "popular": "Popular",
+    "noresults": "Không tìm thấy thông tin bạn cần?",
+    "noresultshelp": "Hãy làm cho JustDeleteMe trở nên tốt hơn",
+    "popular": "Phổ biến",
     "pullrequest": "Gia nhập dự án trên GitHub",
```

Comment:
This became `"pullrequest": "Make a Pull Request on GitHub"` on the source language, can you confirm that it was already updated, please?